nixos/modules/services/web-apps/mobilizon.nix

```diff
@@ -20,7 +28,7 @@ let
   # Make a package containing launchers with the correct envirenment, instead of
   # setting it with systemd services, so that the user can also use them without
   # troubles
-  launchers = pkgs.stdenv.mkDerivation rec {
+  launchers = pkgs.stdenv.mkDerivation {
```

Comment:
Since there is no build phase, could `stdenvNoCC` could be used here?

Reply:
This could just use `runCommand`, right?

Reply:
Thx, addressed in 16fda46db7f39b608388a222f4072708c80eca28